src/mn_font.c

```diff
@@ -0,0 +1,213 @@
+//
+// Copyright(C) 2024 Roman Fomin
```

Comment:
Is this all your own code?

Reply:
I started with the SLADE function, but modified it a lot.

Reply:
Is this the right way?https://github.com/fabiangreffrath/woof/pull/1575/commits/57875dd5b6fa5d93b8d4d66dffe560dd45000922

Reply:
If slade is also licensed under gpl-2, then yes. Could you also add him to the copyright holders in README.md, please?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1575/commits/bd36d3cdc3fcb2831a9f2612b56f5e7ec1062be0

Reply:
Thanks, but could you move the copyright line into the `Files: *` block, please? In general, if code is under the same license and doesn't work as a stand-alone module without our own code, then it should go there.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1575/commits/154423e82d12cd20221b09a991102352dc87adff